modules/rules_cc/metadata.json

```diff
@@ -25,5 +25,7 @@
         "0.0.17",
         "0.1.0"
     ],
-    "yanked_versions": {}
+    "yanked_versions": {
+        "1.1.0": "0.0.15"
```

Comment:
The value should be an actionable user-facing error message.

Reply:
Ah, I see. I've only read the second half of https://github.com/bazelbuild/bazel-central-registry/discussions/128